tests/Functional/Driver/OCI8/ConnectionTest.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\DBAL\Tests\Functional\Driver\OCI8;
+
+use Doctrine\DBAL\DriverManager;
+use Doctrine\DBAL\Exception\DriverException;
+use Doctrine\DBAL\Tests\FunctionalTestCase;
+use Doctrine\DBAL\Tests\TestUtil;
+use PHPUnit\Framework\Attributes\RequiresPhpExtension;
+use Throwable;
+
+#[RequiresPhpExtension('oci8')]
+class ConnectionTest extends FunctionalTestCase
+{
+    private const DISCONNECT_SESSION_SQL = <<<'SQL'
+        DECLARE
+            v_sid    VARCHAR2(10) := :sid;
+            v_serial VARCHAR2(10) := :serial;
+            v_sql    VARCHAR2(200);
+        BEGIN
+            v_sql := 'ALTER SYSTEM DISCONNECT SESSION ''' || v_sid || ',' || v_serial || ''' IMMEDIATE';
```

Comment:
This isn't really better than the original version in terms of security. If binding parameters is impossible, please quote the strings using `Connection::quote()` before using them in the statement.

Reply:
Done, 

This is actually a single string parameter, so it has to be quoted in the form 'SID, SERIAL#'